analyzers/tests/SonarAnalyzer.TestFramework/Verification/Verifier.cs

```diff
@@ -363,5 +369,23 @@ private void ValidateCodeFix()
             && (extension.Equals(".razor", StringComparison.OrdinalIgnoreCase) || extension.Equals(".cshtml", StringComparison.OrdinalIgnoreCase));
 
         public sealed record CompilationData(Compilation Compilation, string[] AdditionalSourceFiles);
+
+        private static int RestorePackages(string path, string workingDirectory)
```

Comment:
What about returning `bool` here? `var value = RestorePackages()` doesn't tell much about what the value means. And we don't need to care.